assets/app/view/spreadsheet.rb

```diff
@@ -96,10 +97,20 @@ def render_title
     end
 
     def render_corporations
-      @game.corporations.map { |c| render_corporation(c) }
+      current_round = [@game.turn]
+      current_round += [@game.round.round_num] if @game.round.name == 'Operating Round'
+
+      floated_corporations = @game.round.entities
+      @game.corporations.map do |c|
+        operating_order = -1
+        operating_index = floated_corporations.find_index(c)
+        operating_order = operating_index + 1 if operating_index
```

Comment:
operating_order = (floated_corporations.find_index(c) || -1) + 1

Reply:
Sweet, will refactor! Gotta keep that C++ from crawling out :)